src/collections/lookup-map.ts

```diff
@@ -1,45 +1,45 @@
 import * as near from '../api'
-import { Bytes } from '../utils';
+import { u8ArrayToLatin1 } from '../utils';
 
 export class LookupMap {
-    readonly keyPrefix: Bytes;
+    readonly keyPrefix: string;
 
-    constructor(keyPrefix: Bytes) {
+    constructor(keyPrefix: string) {
         this.keyPrefix = keyPrefix
     }
 
-    containsKey(key: Bytes): boolean {
+    containsKey(key: string): boolean {
         let storageKey = this.keyPrefix + JSON.stringify(key)
         return near.storageHasKey(storageKey)
     }
 
-    get(key: Bytes): unknown | null {
+    get(key: string): unknown | null {
         let storageKey = this.keyPrefix + JSON.stringify(key)
         let raw = near.storageRead(storageKey)
         if (raw !== null) {
-            return JSON.parse(raw)
+            return JSON.parse(u8ArrayToLatin1(raw))
         }
         return null
     }
 
-    remove(key: Bytes): unknown | null {
+    remove(key: string): unknown | null {
         let storageKey = this.keyPrefix + JSON.stringify(key)
         if (near.storageRemove(storageKey)) {
-            return JSON.parse(near.storageGetEvicted())
+            return JSON.parse(u8ArrayToLatin1(near.storageGetEvicted()))
```

Comment:
Can we localize conversion such conversions in `api.ts`?

Reply:
It depends on a few decisions. 
The complexity is arise from a few places:
- should bytes be alias to string, alias to Uint8Array, or a polymorphic type
- what should the `storage*` returns, string or Uint8Array
- if it returns `Uint8Array`, what should auto-deserialization do, `Uint8Array -> string -> JSON.parse`, or a binary-format deserialization?